endpointmanager/pkg/historypruning/historypruning_integration_test.go

```diff
@@ -726,10 +1016,31 @@ func Test_PruneInfoHistory(t *testing.T) {
 
 	testFhirEndpointInfo.SMARTResponse = nil
 
+	// Check validations
+	err = checkValidationCount(ctx, store, valRes1, 1)
+	th.Assert(t, err == nil, err)
+	err = checkValidationCount(ctx, store, valRes2, 0)
+	th.Assert(t, err == nil, err)
+	err = checkValidationCount(ctx, store, valRes3, 1)
+	th.Assert(t, err == nil, err)
+	err = checkValidationCount(ctx, store, valRes4, 0)
+	th.Assert(t, err == nil, err)
+	err = checkValidationCount(ctx, store, valRes5, 1)
+	th.Assert(t, err == nil, err)
+	err = checkValidationCount(ctx, store, valRes6, 0)
+	th.Assert(t, err == nil, err)
+	err = checkValidationResultCount(ctx, store, 3)
+	th.Assert(t, err == nil, err)
```

Comment:
I just realized there should probably be at least one test where we use a test validation object that has more than one validation, just so that it checks to make sure that if the entry is pruned, all validations for that entry get deleted.